lighthouse-cli/bin.ts

```diff
@@ -260,25 +260,31 @@ function runLighthouse(url: string,
     .then(chrome => chromeLauncher = chrome)
     .then(() => lighthouse(url, flags, config))
     .then((results: Results) => {
+      let promise = Promise.resolve(results);
```

Comment:
maybe move all this printing stuff to its own function? Definitely getting hard to follow :)